ci/tasks/parse-release-tag.yml

```diff
@@ -12,6 +12,6 @@ run:
   args:
     - -ec
     - |
-      RELEASE_NUMBER=$(sed 's/alpha_release-//' < git-release/.git/ref)
-      echo "${RELEASE_NUMBER}-release" > tags/tags
+      RELEASE_NUMBER=$(cut -f 2 -d "-" < git-release/.git/ref)
+      echo "${RELEASE_NUMBER}-release" | tee tags/tags
       echo "${RELEASE_NUMBER}-candidate" > tags/candidate-tag
```

Comment:
Just curious! `tee` is awesome, but why only on one of these and not both of them?

Reply:
Just because they have the same prefix basically, it just felt redundant, but then they do form a different tag, so maybe it would be better on both 🤔 

Yeah good shout, it makes less sense in my other where its saving the number to a file and the tag, but in this it's two different tags!

Reply:
Updated :) Thanks :)